shuttle/static/assets/css/shuttle.css

```diff
@@ -12,6 +12,22 @@ body {
     cursor: auto;
 }
 
+.wrapper {
+    width: 100%;
+    margin-bottom: 20px;
+    display: flex;
+    align-items: center;
+    justify-content: center;
+}
+
+.dropdown{
+    margin: 10px;
+}
+
+.submit{
```

Comment:
Make sure to stay consistent with spacing (ie having a space between the css class/id and the {